CMakeLists.txt

```
@@ -28,7 +28,10 @@ endif(APPLE)
 
 include(GNUInstallDirs)
 
-find_package(podio 0.15 REQUIRED)
+find_package(podio 0.15)
```

Could we use this newer syntax?
```suggestion
find_package(podio 0.15...1.0)
```
See https://cmake.org/cmake/help/latest/command/find_package.html#basic-signature, as of cmake 3.19.

I think upgrading to 3.19 as a requirement is reasonable, since we already need that on mac anyway:
https://github.com/root-project/root/blob/c33e6a4d897ac1dfe7a80c9985c485d90d09be07/CMakeLists.txt#L8

---

See the first commit of the PR (it doesn't work).

---

I guess that makes sense, in hindsight.